File: bloomstack_core/public/js/driver.js
// Copyright (c) 2017, Frappe Technologies Pvt. Ltd. and contributors
// For license information, please see license.txt

frappe.ui.form.on('Driver', {
	user_id: function(frm) {
		if(frm.doc.user_id) {
			frappe.db.get_value("Employee", {"user_id": frm.doc.user_id}, "name", (r) => {
				if(r && r.name) {
					frm.set_value("employee", r.name);
				}	
			});
		}
	}
});

File: bloomstack_core/hook_events/driver.py
# -*- coding: utf-8 -*-
# Copyright (c) 2018, Bloom Stack and contributors
# For license information, please see license.txt

import frappe

def get_employee_from_user(driver, method):
	if method == "validate":
		if driver.user_id:
			employee = frappe.db.get_value("Employee", {"user_id": driver.user_id})
			if employee:
				driver.employee = employee
